src/main/java/org/gridsuite/shortcircuit/server/ShortCircuitController.java

```diff
@@ -105,9 +104,11 @@ public ResponseEntity<Page<FaultResult>> getPagedFaultResults(@Parameter(descrip
                                                                   @Parameter(description = "BASIC (faults without limits and feeders), " +
                                                                       "FULL (faults with both), " +
                                                                       "WITH_LIMIT_VIOLATIONS (like FULL but only those with limit violations) or " +
-                                                                      "NONE (no fault)") @RequestParam(name = "mode", required = false, defaultValue = "WITH_LIMIT_VIOLATIONS") FaultResultsMode mode,
-                                                                  Pageable pageable) {
-        Page<FaultResult> faultResultsPage = shortCircuitService.getFaultResultsPage(resultUuid, mode, pageable);
+                                                                      "NONE (no fault)") @RequestParam(name = "mode", required = false, defaultValue = "FULL") FaultResultsMode mode,
```

Comment:
what does `mode` mean here?

Reply:
But yeah the "NONE" has non sense I agree. I just added some comments.